tests/utils/verification/volumeproperties.go

```diff
@@ -29,10 +29,6 @@ import (
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
-const (
```

Comment:
It is better to have constant to modify the attempt count. I would suggest to add this back; it will be easier to adjust in the future instead of skimming line of code.